java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -65,6 +65,12 @@ public interface TemplateConstants {
   String HIVE_TO_BQ_APPEND_MODE = "hivetobq.write.mode";
   String HIVE_TO_BQ_TEMP_GCS_BUCKET = "hivetobq.temp.gcs.bucket";
 
+  /** Property values for HbaseToGCS */
+  String HBASE_TO_GCS_FILE_FORMAT="hbasetogcs.fileformat";
```

Comment:
property names should denote if they meant for input or output.
so hbasetogcs.fileformat should be either hbasetogcs.gcs.fileformat or hbasetogcs.output.fileformat
Rest of other should follow similar